homeassistant/components/zwave_js/config_flow.py

```diff
@@ -1154,14 +1199,17 @@ async def async_step_finish_addon_setup_migrate(
         assert ws_address is not None
         version_info = self.version_info
         assert version_info is not None
+        config_entry = self._reconfigure_config_entry
+        assert config_entry is not None
 
         # We need to wait for the config entry to be reloaded,
         # before restoring the backup.
         # We will do this in the restore nvm progress task,
         # to get a nicer user experience.
-        self._async_update_entry(
-            {
-                "unique_id": str(version_info.home_id),
```

Comment:
This was another bug found. Instead of updating the unique id we updated the config entry data with a unique id item.